Source/Entities/Validation/ValidationRules.cst

```diff
@@ -87,6 +94,27 @@ namespace <%=NameSpace%>.Validation
             return _rulesList;
          }
       }
+	  
+	  /// <summary>
+	  /// List of common rules associated with an entity.
+	  /// </summary>
+	  private static Dictionary<string, Dictionary<string, List<ValidationRuleInfo>>> CommonRulesList

```

Comment:
The introduction of this means that the property RulesList above no longer returns all the rules.  I assume this may be a breaking change.

Looking for input here,